man/org.freedesktop.systemd1.xml

```diff
@@ -2568,7 +2568,7 @@ node /org/freedesktop/systemd1/unit/avahi_2ddaemon_2eservice {
       @org.freedesktop.DBus.Property.EmitsChangedSignal("const")
       readonly t RestartUSecMax = ...;
       @org.freedesktop.DBus.Property.EmitsChangedSignal("false")
-      readonly t RestartUSecCurrent = ...;
+      readonly t RestartUSecNext = ...;
```

Comment:
This is a public interface, so we cannot change it - there needs to be a new one added instead

Reply:
never mind, it's a new interface